doc/firewalld_services.md

```diff
@@ -0,0 +1,83 @@
+
+
+## Open / Modify Firewall services
+
+A [firewalld service](http://www.firewalld.org/documentation/man-pages/firewalld.service.html) defines a set of ports, protocols and destination addresses simplyfing the process of allow them in a specific zone. 
```

Comment:
"the process of *allow* them..." (?)

Reply:
simplyfying

Reply:
It is easiest to define a service and enable the service per zone than having to add each port explicitly to the zone where you need to open it.